### DISCUSSION TITLE
LEDを一部だけ実装するには

### DISCUSSION THREAD

**Discussion by sugo3** [Yowkees/keyball#332] [Q&A] [answered]
過去のLEDに関する質問を見て調べたので、以下の認識が合っているのか確認したく、教えていただければ幸いです。
全てkeyball39前提です。

1番2番だけ光らせたい場合、その2つを普通にはんだ付けするだけで良い。

途中のLEDを飛ばして実装したい場合、例えば5〜10番を使用しないとしたら、LEDあり4番のDOUTとLEDあり11番のDINをショートさせれば良い。
(LEDなし5番のDINとLEDなし10番のDOUTでも良い)

トラックボールなし側の最後の2つ、23番24番だけ光らせたい場合、LEDなし1番のDINとLEDあり23番のDINをショートさせれば良い。
(1番ではなくもっと大元からいける？)

LED実装のハードルを下げたいので、以上3点の認識があっているのか教えていただければ助かります。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

例を出していただきました３パターンとも正解で、いずれもＬＥＤ点灯させることができます。

最後のパターンはProMicroのLED端子から信号をとることもできますが、LED1のDINから接続で問題ありません。

以上、ご検討よろしくお願いいたします。

**Comment by sugo3** [comment]
早急なご回答ありがとうございます。
これからもkeyball使用させていただきます！

**Reply by Yowkees** [reply]
ありがとうございます！ぜひ末永くご愛用いただければ幸いです。
